test/functoria/e2e/help.t

```diff
@@ -10,7 +10,7 @@ Test that the help command works without config file:
          test-help - Display help about test commands.
   
   SYNOPSIS
-         test help [OPTION]... [TOPIC]
+         test help [--man-format=FMT] [OPTION]… [TOPIC]
```

Comment:
This looks a bit strange. The reason is the following. 

One of the changes that occured by changing the defaults is that now `--help` and `--version` are automatically put in the `COMMON OPTIONS` section. So that the `OPTIONS` you have by default stuff, that is related to the functionality of the tool. If the options listed in the `OPTIONS` seciton are few options (not more than three) the options get listed in synopses. 

In this case I don't think it make sense to make `--man-format` proeminent. So you should set its `docs` argument to `Manpage.s_common_options`.